src/cgame/cg_draw.c

```diff
@@ -4536,7 +4537,7 @@ void CG_Coronas(void)
 				toofar = qtrue;
 			}
 			// dot = DotProduct(dir, cg.refdef_current->viewaxis[0]);
-			const float fov = cosf((cg.refdef_current->fov_x / 2) * (float)(M_PI / 180));
```

Comment:
Head's up, opted for not introducing another scope here and removed `const` here instead.